src/oc_trace_sup.erl

```diff
@@ -0,0 +1,52 @@
+%%%------------------------------------------------------------------------
+%% Copyright 2017, OpenCensus Authors
+%% Licensed under the Apache License, Version 2.0 (the "License");
+%% you may not use this file except in compliance with the License.
+%% You may obtain a copy of the License at
+%%
+%% http://www.apache.org/licenses/LICENSE-2.0
+%%
+%% Unless required by applicable law or agreed to in writing, software
+%% distributed under the License is distributed on an "AS IS" BASIS,
+%% WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+%% See the License for the specific language governing permissions and
+%% limitations under the License.
+%%%------------------------------------------------------------------------
+
+-module(oc_trace_sup).
+
+-behaviour(supervisor).
+
+-export([start_link/1, init/1]).
+
+-include("opencensus.hrl").
+
+start_link(Opts) ->
+  supervisor:start_link(?MODULE, Opts).
```

Comment:
No need for named supervisor. Also I want to inject the configuration from the root supervisor instead of fetching data directly in the workers.